doc/whats_new/v1.0.rst

```diff
@@ -288,12 +288,19 @@ Changelog
   :user:`Clifford Akai-Nettey<cliffordEmmanuel>`.
 
 :mod:`sklearn.calibration`
-............................
+..........................
 
 - |Fix| The predict and predict_proba methods of
-  :class:`calibration.CalibratedClassifierCV can now properly be used on
+  :class:`calibration.CalibratedClassifierCV` can now properly be used on
   prefitted pipelines. :pr:`19641` by :user:`Alek Lefebvre <AlekLefebvre>`
 
+:mod:`sklearn.utils`
+....................
+  
+  - |Fix| Fixed a bug in :func:`utils.sparsefuncs.mean_variance_axis` where the
+    precision of the computed variance was very poor when the real variance is
+    exactly zero. :pr:`19766` by :user:`Jérémie du Boisberranger <jeremiedbb>`.
```

Comment:
#19766 was merged, this PR need to be updated against the current `main`.

Maybe also mention #19788 in the changelog that adds the correction.